cspell.html.txt

```diff
@@ -18,6 +18,7 @@ preload
 progressbar
 rsquo
 searchbox
+setlabeltitle
```

Comment:
I think rather than adding this to the HTML dictionary, it might make more sense to ignore it specifically in the MDX files?

I think you should be able to add this to the top of the MDX files. (Might need to double check in Storybook, but think this works for MDX?)

```mdx
{/* cspell:ignore setlabeltitle */}

```

Reply:
i was trying to remember if that notation existed. that's definitely much more preferable!